tpl/resources/resources.go

```diff
@@ -199,6 +204,12 @@ func (ns *Namespace) FromString(targetPathIn, contentIn interface{}) (resource.R
 	return ns.createClient.FromString(targetPath, content)
 }
 
+// FromRemote creates a Resource from a URL.
+// If you provide multiple parts they will be joined together to the final URL.
+func (ns *Namespace) FromRemote(args ...interface{}) (resource.Resource, error) {
```

Comment:
Remove `ns.FromRemove` and call the client directly (we do not want two template methods doing just about the same thing).

Reply:
Removed the `resources.FromRemote` call, extended test coverage and updated documentation.